utils/linting/license-check-and-add.config.json

```diff
@@ -1,6 +1,6 @@
 {
   "license": "./utils/headers/StrimziHeader.txt",
-  "ignore": [".**", ".git/**", "./utils/headers", "**.md"],
+  "ignore": [".**", ".git/**", "./utils/headers", "**.md", "**.pptx"],
```

Comment:
PDF instead of PPT? For open source purposes

Reply:
The diagrams I have provided thus far have been done in powerpoint (editable by open source tools), and the powerpoint file containing them also has been checked in in case of need to regenerate/change the images. Without this change, the file would be currupted by the license tooling.